sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -1375,9 +1374,8 @@ pi_result piEventGetInfo(pi_event, pi_event_info, size_t, void *, size_t *) {
   DIE_NO_IMPLEMENTATION;
 }
 
-pi_result piEventGetProfilingInfo(pi_event Event, pi_profiling_info ParamName,
-                                  size_t ParamValueSize, void *ParamValue,
-                                  size_t *ParamValueSizeRet) {
+pi_result piEventGetProfilingInfo(pi_event, pi_profiling_info, size_t, void *,
+                                  size_t *) {
```

Comment:
Because the function is not documented it may be good to have something like:
```suggestion
pi_result piEventGetProfilingInfo(pi_event, pi_profiling_info, size_t /*ParamValueSize*/, void */*ParamValue*/,
                                  size_t */*ParamValueSizeRet*/) {
```